runner/utils.py

```diff
@@ -283,3 +284,32 @@ def execute_parallel_jobmanagers(method_name: str, job_managers: 'list[job_manag
     # Wait for all threads to finish
     for thread in threads:
         thread.join()
+
+def update_params_with_values_from_keyvault(parameters: dict(), keyvault_client_with_url: tuple()):
+    """
+    Replaces parameters with keyvault placeholder values with the actual value fetched from keyvault
+    
```

Comment:
Maybe you can add a comment about this being used for corona renderer in here.

Reply:
Currently only used for corona renderer, but it is a general solution for passing secrets as template parameter values. Have added an e.g. for the Corona case